src/Symfony/Component/Translation/Translator.php

```diff
@@ -466,11 +466,12 @@ private function getFallbackContent(MessageCatalogue $catalogue)
     private function getCatalogueCachePath($locale)
     {
         $catalogueHash = sha1(serialize(array(
+            'locale' => $this->locale,
```

Comment:
right, but ```locale``` is a catalog identifier and include it in catalogueHash makes difficult to identify catalog file.